pkgs/development/lisp-modules/packages.nix

```diff
@@ -232,13 +232,13 @@ let
 
   prompter = build-asdf-system rec {
     pname = "prompter";
-    version = "0.1.1";
+    version = "20240108-git";
 
     src = pkgs.fetchFromGitHub {
       owner = "atlas-engineer";
       repo = "prompter";
-      rev = version;
-      sha256 = "sha256-A9gIUBj0oUDFGR5aqHz+tdNR6t03LPMrx0n9qM3ACwE=";
+      rev = "7890ed5d02e70aba01ceb964c6ee4f40776e7dc0";
+      sha256 = "sha256-rRKtpSKAqfzvnlC3NQ4840RrlbBUpI4V6uX6p5hRJWQ=";
```

Comment:
```suggestion
      hash = "sha256-rRKtpSKAqfzvnlC3NQ4840RrlbBUpI4V6uX6p5hRJWQ=";
```

Reply:
There are many hash instead of sha256 in lisp-modules/packages. I think a pr to update the whole file is better than only the two in this pr.

Reply:
```
[kirillvr@osaka:~/nixpkgs/pkgs/development/lisp-modules]$ 
rg 'hash =' packages.nix  | wc -l
15

rg 'sha256 =' packages.nix  | wc -l
26
```

right now nixpkgs is gradually migrating towards using `hash`, eg when line is changed in PR and it using SRI notation, might as well change attribute name.

but I will leave it up to you to decide, At some point I might try a tree wide migration :-)